pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/actions/RestartDeclarativePipelineActionTest.java

```diff
@@ -46,6 +52,8 @@
 import org.jenkinsci.plugins.pipeline.modeldefinition.AbstractModelDefTest;
 import org.jenkinsci.plugins.pipeline.modeldefinition.Utils;
 import org.jenkinsci.plugins.pipeline.modeldefinition.causes.RestartDeclarativePipelineCause;
+import org.jenkinsci.plugins.pipeline.modeldefinition.shaded.com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Oh, just refer to these without `org.jenkinsci.plugins.pipeline.modeldefinition.shaded` - it'll pick up the right shading automatically with `mvn install`, but this import will (and does!) fail at compile time.

Reply:
I always got an error with `mvn install`. Like
`incompatible types: org.jenkinsci.plugins.pipeline.modeldefinition.shaded.com.fasterxml.jackson.databind.JsonNode cannot be converted to com.fasterxml.jackson.databind.JsonNode`

Reply:
yes

Reply:
That is really, really weird - if you look at 
https://ci.jenkins.io/job/Plugins/job/pipeline-model-definition-plugin/job/PR-274/17/display/redirect, you can see the compilation error.